### PR TITLE
feat(carousel): fix to carousel keyboard navigation

### PR DESCRIPTION
**Pull request by sendilkumarn**
#232 

@pkozlowski-opensource  Thanks 👍

### CODE REVIEW

**Comment by pkozlowski-opensource** [comment]
Two more things:
* in BS 4 the keyboard navigation is an option behind a flag: http://v4-alpha.getbootstrap.com/components/carousel/#options (true by default). I wonder if we should support this option as well
* please amend the commit message to include `Fixes #232`

File: src/carousel/carousel.ts [hunk 1]
```diff
@@ -53,6 +61,11 @@ export class NgbCarousel implements AfterContentChecked,
   @Input() interval: number = 5000;
   @Input() wrap: boolean = true;
 
+  @HostBinding('tabIndex')
```
Comment (pkozlowski-opensource) [inline src/carousel/carousel.ts]: We don't need `@HostBinding` for constants.... we could achieve the same effect in `host: {tabIndex: 0}`. Having said this I'm not sure that `0` is the best value. Maybe we should `-1` instead? Or ask users to provide the value?

I must say that I don't have enough knowledge on tab indexes to answer this question myself...

Comment (sendilkumarn) [inline src/carousel/carousel.ts]: As per docs,
-1 won't allow you to have sequential focus navigation
0 will make browser takes care of order in sequential  focus navigation
And 1 or any other positive value for enforcing the order 

I think this is why we use tabindex and also for an element to be focusable we need to set tab index focus flag.

Yes we can achieve it with host.

**Comment by sendilkumarn** [comment]
I thinking supporting it better we can add in that also just like wrap.

I will change the text content.

**Comment by pkozlowski-opensource** [comment]
@sendilkumarn seems like something went wrong during the rebase - the second commit is completely broken. You should have only one commit with only your changes.

**Comment by sendilkumarn** [comment]
Merged with master and then rebased. I will close this open another if okay.

**Comment by pkozlowski-opensource** [comment]
> Merged with master and then rebased. I will close this open another if okay.

Whatever works for you

**Pull request closed without merging**